<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>565</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLineEdit" name="msj">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>490</y>
      <width>531</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="enviarMsj">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>490</y>
      <width>80</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Enviar</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>490</y>
      <width>31</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>p</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>571</width>
      <height>421</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>569</width>
       <height>419</height>
      </rect>
     </property>
     <widget class="QListView" name="chatList">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>571</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QScrollArea" name="scrollArea_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>60</y>
      <width>201</width>
      <height>421</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>199</width>
       <height>419</height>
      </rect>
     </property>
     <widget class="QListView" name="listView">
      <property name="geometry">
       <rect>
        <x>0</x>
        <y>0</y>
        <width>201</width>
        <height>421</height>
       </rect>
      </property>
     </widget>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>26</x>
      <y>4</y>
      <width>231</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>CANAL:</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>798</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuServidor">
    <property name="title">
     <string>Servidor</string>
    </property>
    <addaction name="Crear_Canal"/>
   </widget>
   <widget class="QMenu" name="menuUsuari">
    <property name="title">
     <string>Usuario</string>
    </property>
    <addaction name="actionModificar_Perfil"/>
    <addaction name="actionConectarse_a_un_Canal"/>
    <addaction name="actionventanaChat"/>
   </widget>
   <addaction name="menuServidor"/>
   <addaction name="menuUsuari"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionModificar_Perfil">
   <property name="text">
    <string>Modificar Perfil</string>
   </property>
  </action>
  <action name="Crear_Canal">
   <property name="text">
    <string>Crear Canal</string>
   </property>
  </action>
  <action name="actionConectarse_a_un_Canal">
   <property name="text">
    <string>Conectarse a un Canal</string>
   </property>
  </action>
  <action name="actionventanaChat">
   <property name="text">
    <string>ventanaChat</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
